### DISCUSSION TITLE
Multipart extension

### DISCUSSION THREAD

**Discussion by appsforartists** [community/community#14295] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm looking for all the typings (`.d.ts`) for `Array.prototype.splice`.

I expect [`splice extension:d.ts`](https://github.com/microsoft/TypeScript/search?p=2&q=splice+extension%3Ad.ts) to return these results.  It does not.  If I search instead for [`splice extension:ts`](https://github.com/microsoft/TypeScript/search?p=2&q=splice+extension%3Ats), I get the expected results ([src/lib/es5.d.ts](https://github.com/microsoft/TypeScript/blob/ca65a1a05bed7fbc59739548f169b9747be266e0/src/lib/es5.d.ts), [lib/lib.es5.d.ts](https://github.com/microsoft/TypeScript/blob/cec2fda9a53620dc545a2c4d7b0156446ab145b4/lib/lib.es5.d.ts)), along with others that don't include the infixed `.d`.

**Comment by colinwm** [comment, marked as answer]
You should try the new code search: https://cs.github.com/

There, you can use `path:*.d.ts` to do that
